content/docs/started/workstation/minikube-linux.md

```diff
@@ -327,3 +327,16 @@ Below is an example of accessing a Jupyter notebook hosted in a Linux machine re
    localhost:8889
    ```
    
+## TroubleShooting
+### AttributeError when create namespace object
+During execution of `Use Kubeflow fairing to build the docker image and launch a TFJob for training`, you may see the following error:
```

Comment:
I think a link would be useful here, as people may not know where to find the referenced section.

Reply:
Thanks @sarahmaddox 

May I know how to do this? I tried the following, but no luck:
https://github.com/kubeflow/fairing/blob/master/examples/mnist/mnist_e2e_on_prem.ipynb##Use-Kubeflow-fairing-to-build-the-docker-image-and-launch-a-TFJob-for-training

https://github.com/kubeflow/fairing/blob/master/examples/mnist/mnist_e2e_on_prem.ipynb##Use-Kubeflow-fairing-to-build-the-docker-image-and-launch-a-TFJob-for-training

https://github.com/kubeflow/fairing/blob/master/examples/mnist/mnist_e2e_on_prem.ipynb#cell7

Reply:
Ah I see - as a reader, I thought the text was referring to a section of the doc page, but it's in the notebook. Here's a suggestion, to improve the reader's understanding:

When using Kubeflow Fairing to build a Docker image and launch a training job (part of the [notebook](https://github.com/kubeflow/fairing/blob/master/examples/mnist/mnist_e2e_on_prem.ipynb)), you may see the following error: